pkg/container-disk/container-disk.go

```diff
@@ -112,15 +114,20 @@ func GetDiskTargetPartFromLauncherView(volumeIndex int) (string, error) {
 	return "", fmt.Errorf("no supported file disk found for volume with index %d", volumeIndex)
 }
 
-func GetSocketPathFromHostView(vmi *v1.VirtualMachineInstance, volumeIndex int) (string, error) {
-	for podUID, _ := range vmi.Status.ActivePods {
-		basepath := fmt.Sprintf("/pods/%s/volumes/kubernetes.io~empty-dir/container-disks", string(podUID))
-		exists, _ := diskutils.FileExists(basepath)
-		if exists {
-			return filepath.Join(basepath, fmt.Sprintf("disk_%d.sock", volumeIndex)), nil
+// NewSocketPathGetter get the socket pat of a containerDisk. For testing a baseDir
```

Comment:
nit: "path"

Not worth fixing if it appears the same in master

Reply:
it's the same there